include/sys/spa_impl.h

```diff
@@ -297,6 +297,9 @@ struct spa {
 	taskq_t		*spa_prefetch_taskq;	/* Taskq for prefetch threads */
 	uint64_t	spa_multihost;		/* multihost aware (mmp) */
 	mmp_thread_t	spa_mmp;		/* multihost mmp thread */
+#ifdef	_UZFS
```

Comment:
we hadn't touched spa till now.. may be we never got that need.. can we also explore different way? if not possible, this is fine